### PR TITLE
Kim/sto 3769 dirty swan additions

### PR DESCRIPTION
**Pull request by kimkwanka**
### What this PR does

1. Adds `border-radius` utility classes
2. Adds a `p-body` alternative called `p-container`
3. Adds margin, padding and gap "substeps" below 16px
4. Adds white and transparent text and background color classes
5. Adds documentation for `mx` / `my` & `px` / `py`

image

Solves:  
STO-3769 

### Checklist:

- [x] I reviewed my own code
- [ ] The changes align with the designs I received  
  Or give a reason why this does not apply:
- [ ] I have attached screenshot(s), video(s) or gif(s) showing that the solution is working as expected  
  Or give a reason why this does not apply:
- [x] I have updated the task(s) status on Linear
- [ ] All new media is optimized (images, gifs, videos)

### Browser support

My code works in the following browsers:

- [x] Firefox
- [x] Chrome
- [x] Safari
- [x] Edge

### CODE REVIEW

**Review by talo242 — COMMENTED** [review]
@kimkwanka what do you think if we wait for a little to sync with the design team regarding the sizing values?

I think the borders can be an exception because with those you need more precision.

Everything else looks great!

**Comment by kimkwanka** [comment]
> 

I'm fine with that - tbh I'd rather have multiples of 8 everywhere in the designs instead of this right here.

**Review by talo242 — APPROVED** [review]
Sounds good!

File: src/lib/scss/public/colors/default.scss [hunk 1]
```diff
@@ -66,6 +66,8 @@ $ds-yellow-700: #cc9e21;
 $ds-yellow-900: #4a3d10;
 
 $colors: (
+  'white': #fff,
```
[the remark this answers is not in the document]
Comment (kimkwanka) [reply src/lib/scss/public/colors/default.scss:69]: I'm really not sure why you'd want to use these variables as "white" will always be "white", same with "transparent" no need to use a variable here. But for consistency's sake here's the added variables: https://github.com/getPopsure/dirty-swan/commit/0dc7166161e9070a2ef809a171b7d00741b77a72

[the remark this answers is not in the document]
Comment (keremciu) [reply src/lib/scss/public/colors/default.scss:69]: just stake of semantics

**Pull request merged**